numpy/random/_generator.pyx

```diff
@@ -4448,8 +4448,7 @@ cdef class Generator:
                     "`x` isn't a recognized object; `shuffle` is not guaranteed "
```

Comment:
While you are at it.  `x` isn't a recognized object isn't very helpful.  Would be clearer to say `x` is not a subclass of Sequence. Since the latter has meaning?

Reply:
good point changed it to:
```
UserWarning: you are shuffling a 'dict' object which is not a subclass of 'Sequence'; `shuffle` is not guaranteed to behave correctly. E.g., non-numpy array/tensor objects with view semantics may contain duplicates after shuffling
```